lib/sgtn-client/core/config.rb

```diff
@@ -152,6 +164,24 @@ def logger
           Logging.logger
         end
 
+
+        def loader
+          @loader ||= begin
+            config = SgtnClient::Config.configurations[SgtnClient::Config.default_environment]
+
+            loaders = []
+            loaders << SgtnClient::TranslationLoader::SgtnServer.new if config['vip_server']
+            loaders << SgtnClient::TranslationLoader::LocalTranslation.new if config['translation_bundle']
+            raise 'No translation is available!' if loaders.empty? && config['source_bundle'].nil?
```

Comment:
Use a custom error like SgtnError for sgtn-related errors.

Reply:
fixed